src/services/callsService.js

```diff
@@ -32,11 +32,7 @@ import {
  * @return {number} The actual flags based on the available media
```

Comment:
Diverts from original PR
```suggestion
 * @return {Promise<number>} The actual flags based on the available media
```